README.md

```diff
@@ -1,15 +1,10 @@
 # Portfolio
```

Comment:
- [ ] [Optional] You should always add a description to your PR, kindly implement this